xml/System.Text.RegularExpressions/Regex.xml

```diff
@@ -3241,7 +3247,11 @@ Allows an <see cref="T:System.Object" /> to attempt to free resources and perfor
   
  The regular expression pattern for which the <xref:System.Text.RegularExpressions.Regex.Match(System.String,System.Int32,System.Int32)> method searches is defined by the call to one of the <xref:System.Text.RegularExpressions.Regex> class constructors. For more information about the elements that can form a regular expression pattern, see [Regular Expression Language - Quick Reference](/dotnet/standard/base-types/regular-expression-language-quick-reference).  
   
- The <xref:System.Text.RegularExpressions.Regex.Match(System.String,System.Int32,System.Int32)> method searches the portion of `input` defined by the `beginning` and `length` parameters for the regular expression pattern. `beginning` always defines the index of the leftmost character to include in the search, and `length` defines the maximum number of characters to search. Together, they define the range of the search. If the search proceeds from left to right (the default), the regular expression engine searches from the character at index `beginning` to the character at index `beginning` + `length` - 1. If the regular expression engine was instantiated by using the <xref:System.Text.RegularExpressions.RegexOptions.RightToLeft?displayProperty=nameWithType> option so that the search proceeds from right to left, the regular expression engine searches from the character at index `beginning` + `length` - 1 to the character at index `beginning`. This method returns the first match that it finds within this range. You can retrieve subsequent matches by repeatedly calling the returned <xref:System.Text.RegularExpressions.Match> object's <xref:System.Text.RegularExpressions.Match.NextMatch%2A?displayProperty=nameWithType> method.  
+ The <xref:System.Text.RegularExpressions.Regex.Match(System.String,System.Int32,System.Int32)> method searches the portion of `input` defined by the `beginning` and `length` parameters for the regular expression pattern. `beginning` always defines the index of the leftmost character to include in the search, and `length` defines the maximum number of characters to search. Together, they define the range of the search. The behavior is exactly as if the `input` was effectively `input.Substring(beginning, length)` except that the index of any match is counted relative to the start of `input`. This means that any anchors or zero-width assertions at the start or end of the pattern behave as if there is no `input` outside of this range. For example the anchors `^`, `\G`, and `\A` will be satisfied at `beginning` and `$` and `\z` will be satisfied at `beginning - length`.  
```

Comment:
```suggestion
 The <xref:System.Text.RegularExpressions.Regex.Match(System.String,System.Int32,System.Int32)> method searches the portion of `input` defined by the `beginning` and `length` parameters for the regular expression pattern. `beginning` always defines the index of the leftmost character to include in the search, and `length` defines the maximum number of characters to search. Together, they define the range of the search. The behavior is exactly as if the `input` was effectively `input.Substring(beginning, length)` except that the index of any match is counted relative to the start of `input`. This means that any anchors or zero-width assertions at the start or end of the pattern behave as if there is no `input` outside of this range. For example the anchors `^`, `\G`, and `\A` will be satisfied at `beginning` and `$` and `\z` will be satisfied at `beginning + length - 1`.  
```

Reply:
I think?

Reply:
You're right. I don't know where that came from.